### PR TITLE
add Vietnamese language

### PR DESCRIPTION
**Pull request by lytieuphong**

### CODE REVIEW

File: packages/connectkit/src/localizations/locales/vi-VN.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { LocaleProps } from './';
+
+const enUS: LocaleProps = {
```
Comment (egemenkus) [inline packages/connectkit/src/localizations/locales/vi-VN.ts:3]: It must be
`const viVN: LocaleProps = {`

File: packages/connectkit/src/localizations/locales/vi-VN.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  signInWithEthereumScreen_signedOut_p: `Vui lòng ký tin nhắn yêu cầu \n trên ví của bạn để tiếp tục.`,
+  signInWithEthereumScreen_signedOut_button: 'Đăng nhập',
+
+  signInWithEthereumScreen_signedIn_heading: 'Đăng nhập qua Ethereum',
+  signInWithEthereumScreen_signedIn_h1: 'Bạn đã xác nhận thành công.',
+  signInWithEthereumScreen_signedIn_p: `Sau khi Đăng xuất bạn sẽ cần \n xác nhận lại trong làn tiếp theo.`,
+  signInWithEthereumScreen_signedIn_button: 'Đăng xuất',
+};
+
+export default enUS;
```
Comment (egemenkus) [inline packages/connectkit/src/localizations/locales/vi-VN.ts:117]: It must be
`export default viVN;`

**Review by egemenkus — CHANGES_REQUESTED** [review]

File: packages/connectkit/src/localizations/locales/vi-VN.ts (hunk 1 above)
Comment (lytieuphong) [reply packages/connectkit/src/localizations/locales/vi-VN.ts]: Opp, My apology

**Comment by lytieuphong** [comment]
Please review this PR

**Review by lochie — APPROVED** [review]

**Comment by lochie** [comment]
Sorry that this took incredibly long to review, we were waiting for our translation team to have time to go over this and the ticket got lost over time.

Thank you for the translations @lytieuphong!

**Pull request merged**